### PR TITLE
[docs] mobx-persist-store example added

### PR DESCRIPTION
**Pull request by kanzitelli**
Just thought it would be cool to have an example for `mobx` as well :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,25 @@ export const storage: Storage = {
...
+configurePersistable({
+  storage: {
+    setItem: (key, data) => MMKV.set(key, JSON.stringify(data)),
+    getItem: (key) => JSON.parse(MMKV.getString(key) || '{}'),
+    removeItem: (key) => MMKV.delete(key),
+  },
+});
+```
+
+For more information, check out [example app](https://github.com/kanzitelli/rnn-starter).
```
Comment (mrousavy) [inline README.md:169]:
```suggestion
For more information, check out [kanzitelli/rnn-starter](https://github.com/kanzitelli/rnn-starter).
```

**Review by mrousavy — CHANGES_REQUESTED** [review]
Thanks for your PR! Could you rename the line I marked? That would cause confusion, as "the example app" is the one in this repository (`example/`).

Other than that, it looks good 👍

**Comment by kanzitelli** [comment]
done :)

**Comment by mrousavy** [comment]
Does mobx' `getItem` actually work like that? Does it always return an object?

File: README.md [hunk 2]
```diff
@@ -149,6 +149,25 @@ export const storage: Storage = {
...
+If you want to use MMKV with [mobx-persist-store](https://github.com/quarrant/mobx-persist-store), create the following `storage` object:
+
+```ts
+import { MMKV } from 'react-native-mmkv';
+import { configurePersistable } from 'mobx-persist-store';
+
+configurePersistable({
+  storage: {
+    setItem: (key, data) => MMKV.set(key, JSON.stringify(data)),
+    getItem: (key) => JSON.parse(MMKV.getString(key) || '{}'),
```
Comment (mrousavy) [inline README.md:163]:
```suggestion
    getItem: (key) => JSON.parse(MMKV.getString(key) ?? '{}'),
```

**Review by mrousavy — CHANGES_REQUESTED** [review]
oh I didn't see that - can you change the `||` to use `??`?

**Comment by kanzitelli** [comment]
> Does mobx' `getItem` actually work like that? Does it always return an object?

Actually, it also surprised me but this is the only way I could make it work. But I have checked right now on the working app and it seems to be working without `JSON.parse` and `JSON.stringify` is not necessary in `setItem`. Will also change `||` to `??` 👌

**Comment by kanzitelli** [comment]
done :)

**Comment by mrousavy** [comment]
Thank you! 🖤

**Pull request merged**